api/Document.json

```diff
@@ -7660,7 +7660,7 @@
               "version_added": "56"
             },
             "ie": {
-              "version_added": "10"
+              "version_added": false
```

Comment:
Can you add a note to say that while the event handler attribute wasn't supported, the event itself was?

Reply:
Done!  Does the note look good?